File: invesdwin-scripting-parent/invesdwin-scripting-julia/invesdwin-scripting-julia-runtime-contract/src/test/java/de/invesdwin/scripting/julia/runtime/contract/InputsAndResultsTestLong.jl
println("getLong")
if isdefined(Main, :getLong) && !isnothing(getLong)
	error("getLong already defined!")
end
getLong = putLong
println(typeof(getLong))
println(getLong)
if typeof(getLong) != Int64
	error("getLong not Int64!")
end

println("getLongVector")
if isdefined(Main, :getLongVector) && !isnothing(getLongVector)
	error("getLongVector already defined!")
end
getLongVector = putLongVector
println(eltype(getLongVector))
println(getLongVector)
if eltype(getLongVector) != Int64
	error("getLongVector not Int64!")
end

println("getLongVectorAsList")
if isdefined(Main, :getLongVectorAsList) && !isnothing(getLongVectorAsList)
	error("getLongVectorAsList already defined!")
end
getLongVectorAsList = putLongVectorAsList
println(eltype(getLongVectorAsList))
println(getLongVectorAsList)
if eltype(getLongVectorAsList) != Int64
	error("getLongVectorAsList not Int64!")
end

println("getLongMatrix")
if isdefined(Main, :getLongMatrix) && !isnothing(getLongMatrix)
	error("getLongMatrix already defined!")
end
getLongMatrix = putLongMatrix
println(eltype(getLongMatrix))
println(getLongMatrix)
if eltype(getLongMatrix) != Int64
	error("getLongMatrix not Int64!")
end

println("getLongMatrixAsList")
if isdefined(Main, :getLongMatrixAsList) && !isnothing(getLongMatrixAsList)
	error("getLongMatrixAsList already defined!")
end
getLongMatrixAsList = putLongMatrixAsList
println(eltype(getLongMatrixAsList))
println(getLongMatrixAsList)
if eltype(getLongMatrixAsList) != Int64
	error("getLongMatrixAsList not Int64!")
end

File: invesdwin-scripting-parent/invesdwin-scripting-runtime-lua/src/test/java/de/invesdwin/scripting/runtime/lua/tests/InputsAndResultsTestLong.lua
arraysToString = java.method(java.import('de.invesdwin.util.collections.Arrays'),'toString','java.lang.Object')

print("getLong")
if(type(getLong) ~= "nil") then
	error("getLong already defined!")
end
getLong = putLong
getLongType = type(getLong)
print(getLongType)
print(getLong)
if(getLongType ~= "number") then
	error("getLong not Long!")
end

print("getLongVector")
if(type(getLongVector) ~= "nil") then
	error("getLongVector already defined!")
end
getLongVector = putLongVector
getLongVectorType = type(getLongVector[1])
print(getLongVectorType)
print(arraysToString(getLongVector))
if(getLongVectorType ~= "number") then
	error("getLongVector not Long!")
end

print("getLongVectorAsList")
if(type(getLongVectorAsList) ~= "nil") then
	error("getLongVectorAsList already defined!")
end
getLongVectorAsList = putLongVectorAsList
getLongVectorAsListType = type(getLongVectorAsList[1])
print(getLongVectorAsListType)
print(arraysToString(getLongVectorAsList))
if(getLongVectorAsListType ~= "number") then
	error("getLongVectorAsList not Long!")
end

print("getLongMatrix")
if(type(getLongMatrix) ~= "nil") then
	error("getLongMatrix already defined!")
end
getLongMatrix = putLongMatrix
getLongMatrixType = type(getLongMatrix[1][1])
print(getLongMatrixType)
print(arraysToString(getLongMatrix))
if(getLongMatrixType ~= "number") then
	error("getLongMatrix not Long!")
end

print("getLongMatrixAsList")
if(type(getLongMatrixAsList) ~= "nil") then
	error("getLongMatrixAsList already defined!")
end
getLongMatrixAsList = putLongMatrixAsList
getLongMatrixAsListType = type(getLongMatrixAsList[1][1])
print(getLongMatrixAsListType)
print(arraysToString(getLongMatrixAsList))
if(getLongMatrixAsListType ~= "number") then
	error("getLongMatrixAsList not Long!")
end
